index.html

```diff
@@ -51,9 +51,7 @@ <h2 class="hero-msg">Glad to see you!</h2>
         </div>
       </div>
       <div>
-        <a href="#" class="text-connect">
-          <h5>LET'S CONNECT</h5>
-        </a>
+        <a href="#" class="text-connect"> LET'S CONNECT </a>
```

Comment:
- [ ] As requested by the previous reviewer, please change this into a heading (`hx`) as it should not be a link.

Reply:
:heavy_check_mark: